libraries/Matter/examples/MatterDimmableLight/MatterDimmableLight.ino

```diff
@@ -23,6 +23,8 @@ MatterDimmableLight DimmableLight;
 
 // it will keep last OnOff & Brightness state stored, using Preferences
 Preferences lastStatePref;
+#define ON_OFF_PREF_KEY     "lastOnOffState"
+#define BRIGHTNESS_PREF_KEY "lastBrightness"
```

Comment:
using `const char *` here will need to store the strings only once. Maybe you can also lose the "last" prefix from the var names.